src/components/phone/AgentActions/Actions.vue

```diff
@@ -8,7 +8,8 @@
           t.disabled === true ? 'disabled' : ''
         } `"
       >
-        <ccu-icon v-if="t.icon" with-text size="medium" :type="t.icon">
+        <ccu-icon v-if="t.icon" with-text size="medium" :type="t.icon" :title="$t(t.title)"
```

Comment:
You are going to hate me for this...

But the correct accessibility attribute for images/vectors/icons is 'alt' :grimacing: 

So this (and several of similar instances of this) should be:
```suggestion
        <ccu-icon v-if="t.icon" with-text size="medium" :type="t.icon" :alt="$t(t.title)"
```

Reply:
Ignore this comment. I added a "title" prop to the `ccu-icon` component that just passes the title along to the `alt` attribute. However, still see other comment about `img` tags.